examples/all-clusters-app/esp32/main/Kconfig.projbuild

```diff
@@ -120,18 +122,22 @@ menu "PW RPC Debug channel"
 
     config EXAMPLE_UART_RXD
         int "UART RXD pin number"
-        range 0 34 if IDF_TARGET_ESP32
-        default 3
+        range 0 46
```

Comment:
The range for ESP32 is 0 to 34, for ESP32C3 is 0 to 19. Please refer IDF docs and add correct values for the range and default.

Reply:
Done